TKGame/Entity.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework;
+
+namespace TKGame
+{
+    abstract class Entity
+    {
+        protected Texture2D image;
+        protected Color color = Color.White;
```

Comment:
I feel like something like `texture` or `entityTexture` would be a bit more descriptive than just `image`. Is there a purpose to having `color` be set in the abstract class?